src/rest_api.py

```diff
@@ -70,7 +70,9 @@ def recommender():
     if input_json and 'external_request_id' in input_json and input_json['external_request_id']:
         try:
             check_license = request.args.get('check_license', 'false') == 'true'
-            r = RecommendationTask().execute(input_json, check_license=check_license)
+            persist = request.args.get('persist', 'true') == 'true'
+            r = RecommendationTask().execute(input_json, check_license=check_license,
+                                             persist=persist)
```

Comment:
@ravsa logically this will work, but in the function call of `execute`, the persist parameter is placed before check_license, will be good for readability if we can have persist before check_license here as well :)